app/models/user.rb

```diff
@@ -3,5 +3,6 @@ class User < ApplicationRecord
     has_secure_password
     has_many :photos, dependent: :destroy
     validates :display_name, :email, presence: true
+    has_many :comments
```

Comment:
Dependant destroy?, If so, add tests for destroy if/when users are given such a feature